types/node/ts4.0/tslint.json

```diff
@@ -0,0 +1,13 @@
+// TODO: duplicate or extend?
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+      "ban-types": false,
+      "no-empty-interface": false,
+      "no-outside-dependencies": false,
+      "no-single-declare-module": false,
+      "no-unnecessary-class": false,
+      "unified-signatures": false,
+      "no-bad-reference": false
```

Comment:
Without this I had linting errors. I copied this line from `types/node/ts3.6/tslint.json`.

Reply:
Where there a large number of linting errors? Generally its better to disable lint warnings in specific places (though the node definitions have never fully caught up to all of our lint rules.